src/components/ResultPage.vue

```diff
@@ -26,12 +26,28 @@
         See High Scores
       </h3>
     </router-link>
+    <div class="learning-suggestion">
```

Comment:
Add `v-if="answerCount !== amount"` so that `learning-suggestion` is not shown if all the logos were answered correctly.